launch.go

```diff
@@ -248,88 +248,83 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		"sha":        build.SHA,
 	}
 
-	// If no parent build ID, no parent event, and no event meta, skip fetch meta
-	if len(parentBuildIDs) == 0 && event.ParentEventID == 0 && len(event.Meta) == 0 {
```

Comment:
git diff is weird, the only change is remove this if statement, since now we need to write meta regardless.

Reply:
Diff settings -> Hide whitespace changes make it readable

Reply:
ohh that's cool! didn't know about this feature! 🎆